src/main/java/dk/sdu/se_f22/sortingmodule/infrastructure/domain/SortingModuleImpl.java

```diff
@@ -61,6 +62,9 @@ public void setScoring(String scoring) {
 
     @Override
     public SearchHits search() {
+        SearchModuleImpl searchModule = new SearchModuleImpl();
```

Comment:
Can we use the interface instead of the class for the variable datatype? 

https://github.com/LangeHenrik/SDU-SE-F22/blob/main/src/main/java/dk/sdu/se_f22/searchmodule/infrastructure/interfaces/SearchModule.java